apps/deploy-web/src/pages/api/bitbucket/authenticate.ts

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
```

Comment:
I'd say business logic should be abstracted from the handlers into dedicated modules. Would be better if those are classes.